modules/getting-started-cli-load-data-output.adoc

```diff
@@ -1,23 +1,21 @@
 // Module included in the following assemblies:
 //
-// * getting-started/openshift-cli.adoc
+// * tutorials/dev-app-cli.adoc
 
 :_mod-docs-content-type: PROCEDURE
 [id="getting-started-cli-load-data-output_{context}"]
+= Loading data into the database
 
-= Loading data and displaying the national parks map
-
-You deployed the `parksmap` and `nationalparks` applications and then deployed the `mongodb-nationalparks` database. However, no data has been loaded _into_ the database.
+After you have deployed the `mongodb-nationalparks` database, you can load the national park location information into the database.
 
 .Prerequisites
 
-* You must have access to an {product-title} cluster.
-* You must have installed the OpenShift CLI (`oc`).
-* You have a deployed image.
+* You have deployed the `nationalparks` back-end application.
+* You have deployed the `mongodb-nationalparks` database application.
 
 .Procedure
 
-. To load national parks data, enter the following command:
+. Load the national parks data by running the following command:
```

Comment:
```suggestion
* Load the national parks data by running the following command:
```
Because there is only one step in the procedure, I believe we're supposed to use a bullet instead of a number.

Reply:
Good catch!